deploy/openshift-clusters/roles/kcli/kcli-install/tasks/prepare.yml

```diff
@@ -0,0 +1,112 @@
+---
+# Preparation tasks for kcli-install
+
+- name: Clean up existing cluster if force_cleanup is enabled
+  block:
+    - name: Check existing kcli clusters
+      command: kcli list cluster
+      register: existing_clusters
+      failed_when: false
+      changed_when: false
+
+    - name: Delete existing cluster by name
+      command: >
+        kcli delete cluster openshift {{ test_cluster_name }} --yes
+      register: cluster_cleanup_result
+      failed_when: false
+      when: existing_clusters.stdout_lines | select('search', test_cluster_name) | list | length > 0
+
+    - name: Find VMs matching cluster name pattern
+      command: kcli list vm
+      register: all_vms
+      changed_when: false
+
+    - name: Clean up orphaned VMs if they exist
+      command: kcli delete vm {{ item }} --yes
+      register: vm_cleanup_result
+      failed_when: false
+      loop:
+        - "{{ test_cluster_name }}-ctlplane-0"
+        - "{{ test_cluster_name }}-ctlplane-1"
+        - "{{ test_cluster_name }}-arbiter"
+      when: all_vms.stdout_lines | select('search', item) | list | length > 0
+
+    - name: Display cleanup results
+      debug:
+        msg: |
+          Cluster cleanup: {{ cluster_cleanup_result.stdout | default('not attempted') }}
+          VM cleanup: {{ vm_cleanup_result.results | selectattr('stdout', 'defined') | map(attribute='stdout') | list | default(['not needed']) | join(', ') }}
+
+    - name: Wait for cleanup to complete
+      pause:
+        seconds: 30
+  when: force_cleanup
+
+# Authentication file preparation
+- name: Copy pull secret from files directory to remote host
+  copy:
+    src: "{{ pull_secret_path }}"
+    dest: "{{ ansible_user_dir }}/pull-secret.json"
+    mode: '0600'
+
+- name: Copy SSH public key from localhost to remote host
+  copy:
+    src: "~/.ssh/id_ed25519.pub"
+    dest: "{{ ansible_user_dir }}/.ssh/id_ed25519.pub"
+    mode: '0644'
+  when: ssh_key_stat.stat.exists
+
+- name: Display authentication file locations
+  debug:
+    msg: |
+      Using pull secret from: {{ ansible_user_dir }}/pull-secret.json
+      Using SSH key from: {{ ansible_user_dir }}/.ssh/id_ed25519.pub
+
+- name: Create temporary directory for kcli parameters
+  tempfile:
+    state: directory
+    suffix: kcli-params
+  register: temp_dir
+
+- name: Set kcli parameters file path
+  set_fact:
+    kcli_params_file: "{{ temp_dir.path }}/kcli-params.yml"
+
+- name: Set ksushy IP for BMC simulation based on topology
```

Comment:
wouldn't the ksushy IP only be defined when we are deploying a fencing cluster? We don't need sushy for unless I'm mistaken